### DISCUSSION TITLE
Question: Ignoring file/folder in other apps sanbox folder.

### DISCUSSION THREAD

**Discussion by nawdar-md** [MobiusSync/MobiusSync#224] [Q&A]
First of all I apologize it the place isn’t made for question but I really couldn’t figure it out by my self. 

The question is how would you ignore a file/folder in the other apps folders with are being synced? In my case I need to ignore a folder and reading through syncthing docs I still didn’t understand how to do it since I don’t know what should be writing as a directory  path in iOS system.

Thanks for your reading.

**Comment by SimonPickup** [comment]
The ignore pattens should be relative to the top of the folder you are syncing.
